core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -617,7 +617,8 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
     /**
      * Stores cookies sent by the Java server.
      */
-    @Setter @Getter
+    @Setter
+    @Getter
     private Map<String, byte[]> cookies = new Object2ObjectOpenHashMap<>();
```

Comment:
the Getter should be removable since it is already applied to the class level

Reply:
Fixed in latest commit.